### PR TITLE
Feature/secret manager

### PR DESCRIPTION
**Pull request by Viserion77**
this pr adds a new function to get value from the SM service of AWS.
the test was be ignored because use a was SDK, and I don't know test it.

### CODE REVIEW

**Review by marcosalves-bitg — APPROVED** [review]
LGTM

**Comment by Viserion77** [comment]
This pr was been tested on my computer, and this works!

**Review by marcosalves-bitg — APPROVED** [review]
LGTM

**Comment by th3rius** [comment]
But why not use the SDK directly? Is there a functionality you're trying to abstract?

File: src/lambda/lambdaService.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import AWS from 'aws-sdk'
```
Comment (mickaelnn) [inline src/lambda/lambdaService.ts:1]: Não gostei do jeito que está sendo passado isso para o LambdaService, eu não acho que deveriamos ter um if dentro deste invoke, acredito que exista outros meios de tu passar as keys para esse cara sem precisar desse if...

Caso não tenha outro jeito acharia melhor ainda criar um novo invoke com credenciais para não deixar esse IF

**Review by mickaelnn — CHANGES_REQUESTED** [review]

**Comment by alanneves** [comment]
> But why not use the SDK directly? Is there a functionality you're trying to abstract?

He's trying to access a lambda function from another AWS account. He can access if registers Adapcon credentials in the secret manager of AMC and pass it in the request. I don't think this is a good approach

**Review by ricardobaehr — DISMISSED** [review]
LGTM

Oi gente, dentre as alternativas possíveis, guardar uma key dentro do secret manager na conta da AMC é a mais viável.

Porém deve ser uma key com somente permissão de lambda invoke (podemos estudar se para todos os lambdas ou se vamos especificar cada um).

**Comment by th3rius** [comment]
> He's trying to access a lambda function from another AWS account. He can access if registers Adapcon credentials is in the secret manager of AMC and pass it in the request. I don't think this is a good approach

Sure enough — I just didn't get why we need `SecretManager` when it pretty much does the same thing as the AWS SDK. This

```ts
const client = new AWS.SecretsManager()
// The SDK can also return a promise calling `promise`
const secret = await client.getSecretValue({SecretId: id}).promise()
if (!secret) throw new Error("Secret not found")
const {accessKeyId, secretAccessKey} = JSON.parse(secret.SecretString)
```

is much cleaner than creating another level of abstraction we're perfectly fine without. @alanneves @ricardobaehr what do you think?

**Review by mickaelnn — APPROVED** [review]
LGTM

**Review by th3rius — APPROVED** [review]

**Pull request merged**